api/settings/base.py

```diff
@@ -60,6 +60,13 @@
 
 del TEMPLATE_CONTEXT_PROCESSORS[TEMPLATE_CONTEXT_PROCESSORS.index('apps.feedback.context_processors.feedback')]
 
+# Options for Premailer, which inlines the CSS on the fly in email templates and
+# makes all URLs absolute
+PREMAILER_OPTIONS = {"base_url": os.environ.get("PREMAILER_BASE_URL", "https://www.makeaplea.service.gov.uk"),
```

Comment:
The root level is redirected as per the [Service Manual](https://www.gov.uk/service-manual/technology/managing-domain-names#using-robotstxt-and-root-level-redirections). However this config option is used as a base for URLs with paths that will not be served from `www.gov.uk`.
eg. `https://www.makeaplea.service.gov.uk/plea/enter_urn/` does not redirect to `https://www.gov.uk` and `https://www.gov.uk/make-a-plea/plea/enter_urn/` returns a `404`.